sdk/communication/azure-communication-administration/src/main/java/com/azure/communication/administration/implementation/PhoneNumberAdminClientImplBuilder.java

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.communication.administration.implementation;
+
+import com.azure.core.annotation.ServiceClientBuilder;
+import com.azure.core.http.HttpPipeline;
+import com.azure.core.http.HttpPipelineBuilder;
+import com.azure.core.http.policy.CookiePolicy;
+import com.azure.core.http.policy.RetryPolicy;
+import com.azure.core.http.policy.UserAgentPolicy;
+
+/** A builder for creating a new instance of the PhoneNumberAdminClient type. */
+@ServiceClientBuilder(serviceClients = {PhoneNumberAdminClientImpl.class})
+public final class PhoneNumberAdminClientImplBuilder {
+    /*
+     * The endpoint of the Azure Communication resource.
+     */
+    private String endpoint;
+
+    /**
+     * Sets The endpoint of the Azure Communication resource.
+     *
+     * @param endpoint the endpoint value.
+     * @return the PhoneNumberAdminClientImplBuilder.
+     */
+    public PhoneNumberAdminClientImplBuilder endpoint(String endpoint) {
+        this.endpoint = endpoint;
+        return this;
+    }
+
+    /*
+     * The HTTP pipeline to send requests through
+     */
+    private HttpPipeline pipeline;
+
+    /**
+     * Sets The HTTP pipeline to send requests through.
+     *
+     * @param pipeline the pipeline value.
+     * @return the PhoneNumberAdminClientImplBuilder.
+     */
+    public PhoneNumberAdminClientImplBuilder pipeline(HttpPipeline pipeline) {
+        this.pipeline = pipeline;
+        return this;
+    }
+
+    /**
+     * Builds an instance of PhoneNumberAdminClientImpl with the provided parameters.
+     *
+     * @return an instance of PhoneNumberAdminClientImpl.
+     */
+    public PhoneNumberAdminClientImpl buildClient() {
+        if (pipeline == null) {
+            this.pipeline =
+                    new HttpPipelineBuilder()
+                            .policies(new UserAgentPolicy(), new RetryPolicy(), new CookiePolicy())
+                            .build();
```

Comment:
nit: I am not yet aware of the coding guidelines but the 'new' going to next line seems wrong